config/samples/backends/lvm/iscsi/backend.yaml

```diff
@@ -28,3 +26,7 @@ spec:
             target_protocol=iscsi
             target_helper=lioadm
             volume_backend_name=lvm_iscsi
+            # This is the OCP host's static IP on the storage VLAN network
+            target_ip_address=172.18.0.5
```

Comment:
Are these "magic" addresses when using CRC? What about 3-node OCP?

Reply:
Yes, those are for CRC when deploying with `install_yamls`, I think I mention it in the `README.md` file:
- https://github.com/openstack-k8s-operators/install_yamls/blob/17f749afc85155faba76f8fa878c213e7678f159/scripts/gen-nncp.sh#L244
- https://github.com/openstack-k8s-operators/install_yamls/blob/17f749afc85155faba76f8fa878c213e7678f159/scripts/gen-nncp.sh#L289

I don't know about `ci-framework` deployments.